src/event.js

```diff
@@ -0,0 +1,2 @@
+var syn = require('./element');
```

Comment:
Lint error: `src/event.js: line 1, col 5, 'syn' is defined but never used.`

Is this file needed?

Reply:
Not needed. It is my plan to split code up into individual responsibilities, but other than creating the files I haven't done anything with them yet.